### DISCUSSION TITLE
How can I link all traces automatically to an annotation queue

### DISCUSSION THREAD

**Discussion by wisefinger** [langchain-ai/langsmith-sdk#866] [Q&A] [answered]
Hi, In Langsmith I can link a trace to an annotation queue by clicking a button in the UI. But in my case, I want to set up my system so that all my incoming traces are added to the annotation queue. I'm developing a new product and I want to annotate all traces. I was not able to find any documentation on how to configure this, and adding all traced to the queue in the UI is not a really nice option. Any help is much appreciated.

**Comment by bhanson-techempower** [comment, marked as answer]
https://docs.smith.langchain.com/how_to_guides/monitoring/rules
